dimension-reduction/umap_example.Rmd

```diff
@@ -0,0 +1,109 @@
+---
+title: "UMAP Visualization"
+author: "ALSF CCDL - Candace Savonen"
+output:   
+  html_notebook: 
+    toc: true
+    toc_float: true
+---
+
+*Purpose*: This notebook illustrates how to perform Uniform Manifold Approximation
+and plot the coordinates using ggplot2. 
+
+Recommended articles on UMAP:   
+- [How UMAP works with visuals](https://umap-learn.readthedocs.io/en/latest/how_umap_works.html)  
+- [Original paper on UMAP](https://arxiv.org/abs/1802.03426) by McGinnes, Healy & Melville.  
+
+## 1) Set up libraries
+Install [`umap` R package](https://cran.r-project.org/web/packages/umap/vignettes/umap.html) 
+if you have not installed it previously. 
+
+```{r eval = FALSE}
+install.packages("umap")
```

Comment:
Would change to check if in `installed.packages()` and get rid of `eval = FALSE`